extlinks/settings/production.py

```diff
@@ -2,15 +2,16 @@
 
 DEBUG = False
 
-ALLOWED_HOSTS += ['wikilink.wmflabs.org']
+ALLOWED_HOSTS += ["wikilink.wmflabs.org"]
 
 # We can only run this in production, because we're connecting to the database
 # replicas internally.
-CRON_CLASSES += [
-    'extlinks.links.cron.TotalLinksCron'
-]
+CRON_CLASSES += ["extlinks.links.cron.TotalLinksCron"]
 
 # Redirect HTTP to HTTPS
 # SECURE_PROXY_SSL_HEADER is required because we're behind a proxy
 SECURE_SSL_REDIRECT = True
-SECURE_PROXY_SSL_HEADER = ('HTTP_X_FORWARDED_PROTO', 'https')
+SECURE_PROXY_SSL_HEADER = ("HTTP_X_FORWARDED_PROTO", "https")
+
+SERVER_EMAIL = "Wikipedia Library Card Platform <noreply@wikipedialibrary.wmflabs.org>"
```

Comment:
This (and in staging.py) should probably be `Wikilink` instead of `Wikipedia Library Card Platform`?

Reply:
Thanks for the catch! Changed it!